groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/api/completion/CompletionItem.java

```diff
@@ -910,30 +955,18 @@ public boolean equals(Object obj) {
         }
     }
 
-    public static class JavaFieldItem extends CompletionItem {
-
-        private final String className;
-        private final String name;
+    public static class JavaFieldItem extends JavaElementItem {
```

Comment:
API change.

Reply:
@JaroslavTulach  I'll better rework the change - it'd prefer not to introduce any API changes at all :-/ Those APIs are too open already. Sorry for the inconvenience - you'll be reviewing once more ...